packages/snaps-cli/src/cmds/build/bundle.ts

```diff
@@ -22,15 +22,14 @@ export function bundle(
   dest: string,
   argv: YargsArgs,
 ): Promise<boolean> {
-  const { sourceMaps: debug, transpilationMode } = argv;
+  const { sourceMaps: debug, transpilationMode, options } = argv;
 
   return new Promise((resolve, _reject) => {
     const bundleStream = createBundleStream(dest);
     const bundler = browserify(src, { debug });
-
     if (transpilationMode !== TranspilationModes.none) {
       bundler.transform(require('babelify'), {
-        global: transpilationMode === TranspilationModes.all,
+        global: transpilationMode !== TranspilationModes.localOnly,
```

Comment:
This seems more future-proof, in case we add more transpilation modes or refactor this function:

```suggestion
        global: transpilationMode === TranspilationModes.localAndDeps,
```